doc/content/examples/pore_scale_transport/index.md

```diff
@@ -0,0 +1,308 @@
+
+\begin{equation} \label{eq:BC}
+c_{s,BC,pore} = \frac{P}{RT}
+\end{equation}
+
+where $P$ is the tritium pressure (Pa), $R$ is the gas constant from [PhysicalConstants](PhysicalConstants.md), and $T$ is the temperature (K).
+
+!listing test/tests/pore_scale_transport/2D_absorption_base.i link=false block=BCs
+
+As initial conditions, the ceramic initially contains no tritium, and the pore is filled with $T_2$ with the value from Eq. [eq:BC].
+
+!listing test/tests/pore_scale_transport/2D_absorption_base.i link=false block=ICs
+
+!listing test/tests/pore_scale_transport/2D_absorption_base.i link=false block=Functions
+
+#### Run simulations
+
+To run the input file to reproduce the absorption results shown in Fig. 6 in [!cite](Simon2022), users need to first run the previous simulations from the first step. As a result, TMAP8 will have generated `pore_structure_open/pore_structure_open.e` and `pore_structure_close/pore_structure_close.e`. Then, to model tritium transport in the microstructure with closed pores, run:
+
+```
+~/projects/TMAP8/test/tests/pore_scale_transport/
+mpirun -np 4 ~/projects/TMAP8/tmap8-opt -i 2D_absorption_base.i pore_structure_closed_absorption.params
+```
+
+and to model tritium transport in the microstructure with open pores, run:
+
+```
+~/projects/TMAP8/test/tests/pore_scale_transport/
+mpirun -np 4 ~/projects/TMAP8/tmap8-opt -i 2D_absorption_base.i pore_structure_open_absorption.params
+```
+
+Note that [pore_structure_closed_absorption.params](pore_structure_closed_absorption.params) and [pore_structure_open_absorption.params](pore_structure_open_absorption.params) complement the base input file [2D_absorption_base.i](2D_absorption_base.i) by providing input and output names.
+
+### Results and Discussion
+
+!style halign=left
+Figure 6 in [!cite](Simon2022) shows the tritium absorption profiles for samples with different pore interconnectivities. The results indicate that higher pore interconnectivity leads to faster tritium transport, which is the type of insight that one-dimensional simulations cannot provide.
+
+These outputs are obtained via the postprocessor blocks:
+
+!listing test/tests/pore_scale_transport/2D_absorption_base.i link=false block=Postprocessors
+
+Note that these simulations are not run to generate  Fig. 6 from [!cite](Simon2022) on the fly in the documentation like it is custom to do in other [examples](examples/tmap_index.md) and [V&V](verification_and_validation/index.md) cases due to the significant time these simulations can take when using a fine mesh. However, using the instructions and details provided above, users can replicate the results presented in [!cite](Simon2022). If any challenge arise or if you have a question while working through this example case, feel free to reach out to the TMAP8 development team on the [TMAP8 GitHub discussion page](https://github.com/idaholab/TMAP8/discussions).
```

Comment:
While acknowledging that these aren't run on-the-fly, do you think its still useful to show the results to the interested reader?

Reply:
In general, adding results that are not generated on the fly can be a maintenance challenge if input files/models are updated without the documentation being updated.

In this case, I prefer to refer to the results of the paper and to the first figure, which shows the results on the right. However, I don't think we need to reproduce the figure here again.

Reply:
That's fine - I was just wanting to probe your reasoning a bit. I am fine with leaving it out.